repository/process.go

```diff
@@ -18,6 +34,17 @@ func ProcessRepos(gitxargsConfig *config.GitXargsConfig, repos []*github.Reposit
 
 	for _, repo := range repos {
 		wg.Add()
+
+		go func() {
+			for {
+				select {
+				case pr := <-gitxargsConfig.PRChan:
+					wg.Add()
+					go pullRequestWorker(gitxargsConfig, pr, &wg)
```

Comment:
NIT: Do you really need to run in that in a separate goroutine? Could you not call this method directly here, as it uses the ticker internally to space things out?

Reply:
That would work, too, but I liked reusing the same wait group for both goroutines since, that way, the internal work queue naturally grows and shrinks according to the work to be done, and git-xargs returns once all processing is complete.